modules/web-crypto-backend/test/backend-factory.test.ts

```diff
@@ -189,3 +195,25 @@ describe('webCryptoBackendFactory', () => {
     })
   })
 })
+
+describe('getNonZeroByteBackend', () => {
+  it('gets a subtle backend', () => {
+    const test = getNonZeroByteBackend(fixtures.subtleFallbackSupportsZeroByteGCM)
+    expect(test === fixtures.subtleFallbackSupportsZeroByteGCM.subtle).to.equal(true)
+  })
+
+  it('Precondition: A backend must be passed to get a non zero byte backend.', () => {
+    expect(() => getNonZeroByteBackend(false)).to.throw('No supported backend.')
```

Comment:
Is there a convenient way to make the meaning of the `false` more intuitive to the reader (e.g. by naming it, or adding a comment for what it does)?

Reply:
Hum... These functions operate on backends returned.  I can add a comment.  But the idea is that if a backend does not exist, I return false to be explicit.  These helper functions are just handling that...

Reply:
We chatted about this offline and I can't think of a non-confusing way to express what I want here, so I would say let's just leave it.